crates/nargo_cli/src/cli/init_cmd.rs

```diff
@@ -37,6 +37,13 @@ fn test_main() {
 }
 "#;
 
+const CONTRACT_EXAMPLE: &str = r#"contract Main {
+    fn double(x: Field) -> pub Field { x * 2 }
+    fn triple(x: Field) -> pub Field { x * 3 }
+    internal fn quadruple(x: Field) -> pub Field { x * 4 }
+}
+"#;
```

Comment:
Do we have a better example contract to init?

Reply:
Maybe make `double` internal rather than `quadruple` and implement it in terms of `double`?

Reply:
Great suggestion. Done!